### PR TITLE
[13.0][MIG] mail_tracking_mailgun

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Standard migration including b96742ecfa5b73966c1e779413e1b36aa25a584f

CC @ForgeFlow @AaronHForgeFlow @pedrobaeza

### CODE REVIEW

**Review by alan196 — APPROVED** [review]
Tested in a production environment 👍

**Comment by pedrobaeza** [comment]
Please rebase undoing the changes in isort.cfg and I'm afraid you will need to format again code. My advice on the steps to follow are:

- Save current module folder with all migrated.
- Go back to the black, isort commit.
- Rebase
- Run pre-commit again with latest definition and commit amending existing commit.
- Delete module folder and copy back your folder backup.
- Commit the existing diff.

**Comment by HviorForgeFlow** [comment]
I think this is not required as I squash new precommit changes into black, isort commit. Precommit does not format strings properly, so in migration commit I did the fix.

**Comment by pedrobaeza** [comment]
No, that's not correct, you will be squashing that auto-formatting stuff with the migration stuff, and that's impossible to be reviewed. Please proceed as I said.

File: mail_tracking_mailgun/__manifest__.py [hunk 1]
```diff
@@ -6,13 +6,14 @@
...
+    "version": "13.0.1.0.0",
     "category": "Social Network",
     "website": "https://github.com/OCA/social",
-    "author": "Tecnativa, " "Odoo Community Association (OCA)",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
     "license": "AGPL-3",
     "application": False,
     "installable": True,
     "depends": ["mail_tracking"],
+    "external_dependencies": {"python": ["requests"]},
```
Comment (pedrobaeza) [inline mail_tracking_mailgun/__manifest__.py]: This is not needed for example

**Comment by pedrobaeza** [comment]
Note also that this is not failing as it doesn't contain the rebase. When rebasing, you'll find all the new auto-formatting thing (in XML, JS, etc).

**Comment by HviorForgeFlow** [comment]
Comments attended

**Comment by pedrobaeza** [comment]
Have you rebased over 13.0 branch?

**Comment by HviorForgeFlow** [comment]
Yes, in deed!

**Comment by pedrobaeza** [comment]
Let's see if everything goes well:

/ocabot merge

**Comment by pedrobaeza** [comment]
It has worked! It seems there are not XML changes made by prettier.

**Pull request merged**